runtime/shared_common/shrinit.cpp

```diff
@@ -255,6 +255,11 @@ J9SharedClassesHelpText J9SHAREDCLASSESHELPTEXT[] = {
 	{HELPTEXT_ADJUST_MAXAOT_EQUALS, J9NLS_SHRC_SHRINIT_HELPTEXT_ADJUST_MAXAOT_EQUALS, 0, 0},
 	{HELPTEXT_ADJUST_MINJITDATA_EQUALS, J9NLS_SHRC_SHRINIT_HELPTEXT_ADJUST_MINJIT_EQUALS, 0, 0},
 	{HELPTEXT_ADJUST_MAXJITDATA_EQUALS, J9NLS_SHRC_SHRINIT_HELPTEXT_ADJUST_MAXJIT_EQUALS, 0, 0},
+#if defined(J9VM_OPT_MULTI_LAYER_SHARED_CLASS_CACHE)
```

Comment:
I think moving the options to the help should wait until the remaining layer support and testing is delivered. We'll target the 0.18 release.

Reply:
Removed this change.